java/ql/lib/change-notes/2023-09-28-moved-localuserinput-to-mad.md

```diff
@@ -0,0 +1,20 @@
+---
+category: minorAnalysis
+---
+* Modified the `EnvInput` class in `semmle.code.java.dataflow.FlowSources` to include `environment` and `file` source nodes.
```

Comment:
Maybe it is worth mentioning that the change is semantics preserving unless someone has added their own MaD models with these kinds.